src/test/java/seedu/multitasky/model/EntryBookTest.java

```diff
@@ -0,0 +1,132 @@
+package seedu.multitasky.model;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.util.List;
+
+import org.junit.Test;
+
+import seedu.multitasky.model.entry.Deadline;
+import seedu.multitasky.model.entry.DeadlineList;
+import seedu.multitasky.model.entry.DeadlineListTest;
+import seedu.multitasky.model.entry.Entry;
+import seedu.multitasky.model.entry.Event;
+import seedu.multitasky.model.entry.EventList;
+import seedu.multitasky.model.entry.EventListTest;
+import seedu.multitasky.model.entry.FloatingTask;
+import seedu.multitasky.model.entry.FloatingTaskList;
+import seedu.multitasky.model.entry.FloatingTaskListTest;
+import seedu.multitasky.model.entry.ReadOnlyEntry;
+import seedu.multitasky.model.entry.exceptions.DuplicateEntryException;
+import seedu.multitasky.model.entry.exceptions.EntryNotFoundException;
+
+public class EntryBookTest {
+
+    private EventList[] listOfEventLists = EventListTest.getListOfSampleEventLists();
+    private DeadlineList[] listOfDeadlineLists = DeadlineListTest.getListOfSampleDeadlineLists();
+    private FloatingTaskList[] listOfFloatingTaskLists = FloatingTaskListTest.getListOfSampleFloatingTaskLists();
+
+    // @@author A0126623L
+    /**
+     * @return EntryBook    with sample events, deadlines and floating tasks.
+     * @throws DuplicateEntryException
+     */
+    public static EntryBook getSampleEntryBook() throws DuplicateEntryException {
+
+        EventList[] listOfEventLists = EventListTest.getListOfSampleEventLists();
+        DeadlineList[] listOfDeadlineLists = DeadlineListTest.getListOfSampleDeadlineLists();
+        FloatingTaskList[] listOfFloatingTaskLists = FloatingTaskListTest.getListOfSampleFloatingTaskLists();
+
+        EntryBook entryBook = new EntryBook();
+        for (Entry event : listOfEventLists[0]) {
+            assertTrue(event instanceof Event);
+            entryBook.addEntry(event);
+        }
+
+        for (Entry event : listOfDeadlineLists[0]) {
+            assertTrue(event instanceof Deadline);
+            entryBook.addEntry(event);
+        }
+
+        for (Entry event : listOfFloatingTaskLists[0]) {
+            assertTrue(event instanceof FloatingTask);
+            entryBook.addEntry(event);
+        }
+
+        return entryBook;
+    }
+    // @@author
```

Comment:
I'm doing this so that in case anyone inserts more test cases in the middle of this file in the future, it'll be easier for them. They won't need to reinsert my tags below the test methods they inserted.